src/Components/benchmarkapps/Wasm.Performance/TestApp/wwwroot/benchmarks/grid.js

```diff
@@ -0,0 +1,105 @@
+import { group, benchmark, setup, teardown } from './lib/minibench/minibench.js';
+import { receiveEvent } from './util/BenchmarkEvents.js';
+import { BlazorApp } from './util/BlazorApp.js';
+import { setInputValue } from './util/DOM.js';
+
+group('Grid', () => {
+  let app;
+
+  setup(async () => {
+    app = new BlazorApp();
+    await app.start();
+    app.navigateTo('gridrendering');
+  });
+
+  teardown(() => {
+    app.dispose();
+  });
+
+  benchmark('PlainTable: From blank', () => measureRenderGridFromBlank(app), {
```

Comment:
Filed https://github.com/dotnet/aspnetcore/issues/23303 to track adding these to the benchmarks UI